choco/docker-machine-vmwareworkstation.nuspec

```diff
@@ -0,0 +1,29 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>docker-machine-vmwareworkstation</id>
+    <title>docker-machine-vmwareworkstation</title>
+    <version>1.0.0</version>
+    <authors>Gonzalo Peci</authors>
+    <owners>StefanScherer</owners>
```

Comment:
As im reading from the nuspec Gonzalo Peci should be the owner and both of us should be the authors i think.